dockerfiles/BioBamBam2.0.106/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:bionic-20191029
+
+RUN apt-get update \
+ && apt-get install -y \
+ tar \
+ gzip \
+ wget \
+ xz-utils
+
+WORKDIR /tmp
+
+#Get biobambam2
+RUN wget https://gitlab.com/german.tischler/biobambam2/uploads/8006c668e1fad2875f1fd2d952a502fb/biobambam2_x86_64-linux-gnu_2.0.106.tar.xz
```

Comment:
slight preference to use a variable for `biobambam2_x86_64-linux-gnu_2.0.106.tar.xz` here. Makes C&P errors more likely with future maintenance

Reply:
done